textcat/textcat_correct.py

```diff
@@ -0,0 +1,115 @@
+import copy
+from typing import List, Optional
+import prodigy
+from prodigy.components.loaders import JSONL
+from prodigy.util import split_string
+import spacy
+from spacy.tokens import Doc
+from spacy.training import Example
+
+
+# Recipe decorator with argument annotations: (description, argument type,
+# shortcut, type / converter function called on value before it's passed to
+# the function). Descriptions are also shown when typing --help.
+@prodigy.recipe(
+    "textcat.correct",
+    dataset=("The dataset to use", "positional", None, str),
+    spacy_model=("The base model", "positional", None, str),
+    source=("The source data as a JSONL file", "positional", None, str),
+    label=("One or more comma-separated labels", "option", "l", split_string),
+    update=("Whether to update the model during annotation", "flag", "UP", bool),
+    exclude=("Names of datasets to exclude", "option", "e", split_string),
+    threshold=("Score threshold to pre-select label", "option", "t", float),
+    component=("Name of text classifier component in the pipeline \
+                (will be guessed from pipeline if not set)", "option", "c", str),
```

Comment:
Nitpick: I would generally avoid `\` and just have it in one line (and use `# fmt: off` / `# fmt: on`  to prevent line breaks by auto formatting if it's relevant – probably not here).

```suggestion
    component=("Name of text classifier component in the pipeline (will be guessed from pipeline if not set)", "option", "c", str),
```

Reply:
Fixed as suggested.